tortoise_interpreter.py

```diff
@@ -18,6 +18,8 @@ def main_tortoise(program, print_to_screen=print):
     }
     command_to_parse = navigate_command_functions(commands, program)
 
+    if args.verbose:
```

Comment:
You've added behavior, but not changed the tests. You'll need to test that this works.
Gentle reminder; I taught you to write the tests first, with good reason. Bear that in mind next time :)